src/test/java/SecuritiesManagerTest.java

```diff
@@ -0,0 +1,119 @@
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.util.List;
+
+import org.junit.jupiter.api.Test;
+
+import com.lviv.iot.lab2.Main;
+import com.lviv.iot.lab2.models.DebtSecurities;
+import com.lviv.iot.lab2.models.DebtSecuritiesType;
+import com.lviv.iot.lab2.models.RiskLevel;
+import com.lviv.iot.lab2.models.Securities;
+import com.lviv.iot.lab2.models.StockTrend;
+
+import junit.framework.Assert;
+
+public class SecuritiesManagerTest extends BaseSecuritiesTest {
+	
+	@Test
+	public void testAscendingSortByPrice() {
+		List<Securities> result = securitiesManager.ascendingSortByPrice(testStockMarket);
```

Comment:
It would be better if you named it something like this

> - sortedByPrice
> - sortedByPriceAsc
> - sortedSecuritiesByPriceAsc?  

A lot of ways, but not result

Reply:
Without the word "test" in it, you mean?

Reply:
Just rename '`result`' in some way